.vscode/launch.json

```diff
@@ -89,6 +89,32 @@
             "request": "launch",
             "url": "http://localhost:8080/",
             "webRoot": "${workspaceRoot}"
+        },
+        {
+            // http://blog.likewise.org/2017/02/debugging-a-javascript-webdriverio-project-in-vscode/
+            // http://www.wiktorzychla.com/2017/08/debugging-javascript-webdriver-io-in.html
+            "type": "node",
+            "request": "launch",
+            "port": 5859,
+            "protocol": "inspector",
+            "name": "WebdriverIO",
+            "runtimeExecutable": "${workspaceRoot}/node_modules/.bin/wdio",
+            "windows": {
+                "runtimeExecutable": "${workspaceRoot}/node_modules/.bin/wdio.cmd"
+            },
+            "cwd": "${workspaceRoot}/examples/browser",
+            "console": "integratedTerminal",
+            // This args config runs only the file that's open and displayed
+            // (e.g., a file in test/spec/):
+            "args": [
+                "--spec",
+                "${workspaceRoot}/${relativeFile}"
+                // To run a specific file, you can also do:
+                //"--spec", "test/specs/foo.spec.js"
```

Comment:
would be nice to use SPEC env as [here](http://webdriver.io/guide/testrunner/debugging.html#Dynamic-configuration)